llama_stack/distribution/build.py

```diff
@@ -105,6 +121,11 @@ def build_image(
 
     normal_deps, special_deps = get_provider_dependencies(build_config)
     normal_deps += SERVER_DEPENDENCIES
+    if build_config.external_apis_dir:
+        external_apis = load_external_apis(build_config)
+        if external_apis:
+            for _, api_spec in external_apis.items():
+                normal_deps.extend(api_spec.pip_packages)
```

Comment:
Is it harmful if multiple external APIs share the same dependencies and those entries enter multiple times in normal_deps ? Shouldn't dups in the array be removed ?

Also, what's about version mismatch of dependencies coming from different external API definitions ?

Reply:
I think this could lead to an issue if some providers have incompatible/unresolvable dependencies, but this is not something we can control. We will encounter an error during the build, and actions must be taken with the external API owners to resolve the dependency issues, which are beyond the scope of our code.

The dups are not an issue.